libV2/server/API/internal/prometheusMetrics.js

```diff
@@ -0,0 +1,15 @@
+const { collectDefaultMetrics, register } = require('prom-client');
```

Comment:
this provides nodejs server metric. What other custom metrics at the service level would be useful?

Reply:
It could be nice to have some http level metrics. requests per route/operation, http errors etc.